doc/Makefile

```diff
@@ -158,21 +174,11 @@ doctest:
 	@echo "Testing of doctests in the sources finished, look at the " \
 	      "results in $(BUILDDIR)/doctest/output.txt."
 
-download-data:
-
 zip: html pdf
 	mkdir -p _build/nilearn ;
 	cp -r _build/html _build/nilearn ;
-	cp -r data _build/nilearn ;
-	cp nilearn.pdf _build/nilearn;
 	zip -r _build/nilearn.zip _build/nilearn
 
-pdf: latex
-	cd _build/latex ; make all-pdf ; pdfnup niearn.pdf
-	cp _build/latex/nilearn.pdf nilearn-simple.pdf
-	cp _build/latex/nilearn-nup.pdf nilearn.pdf
-	#cd _build/latex ; make all-pdf ; pdfnup nilearn.pdf
```

Comment:
This assumes that we do not want to publish the doc as pdf beyond what a latex build can give us.